app/support/reservations/date_support.rb

```diff
@@ -73,9 +73,9 @@ def duration_unit
 
   def duration_mins
     if @duration_mins
-      @duration_mins.to_i
+      @duration_mins.to_f.round
     elsif reserve_end_at and reserve_start_at
-      ((reserve_end_at - reserve_start_at) / 60).floor
+      ((reserve_end_at - reserve_start_at) / 60).round
```

Comment:
We actually do want to truncate, not round. It needs to match the logic that we're using for price calculations. See what we're doing in InstrumentPricePolicyCalculations. We want to truncate start and end, and then subtract.

Not sure if it's worth the refactoring of the price calculations to use `duration_mins` or whether that would be too dangerous a change.